public/signup.html

```diff
@@ -36,7 +36,7 @@ <h2>Sign Up Form</h2>
           <button type="submit" class="btn btn-default">Sign Up</button>
         </form>
         <br />
-        <p>Or log in <a href="/login">here</a></p>
+        <p>Or log in <a href="/login.html">here</a></p>
```

Comment:
This should also be left as it is.  it references a route which the htmlroutes then directs to the page